config/locales/views/mailer/hi.yml

```diff
@@ -0,0 +1,39 @@
+en:
+  mailer:
+    welcome_heading: "सर्किटवर्स में आपका स्वागत है, %{name}"
+    keep_building: "निर्माण करते रहे"
+    view_project: "परियोजना देखें"
```

Comment:
I think we should make use of प्रोजेक्ट instead of परियोजना, to make it less complex.

Reply:
We can replace परियोजना to प्रोजेक्ट everywhere in the file.

Reply:
I have used परियोजना from starting in other modules also as suggested by @ShreyaPrasad1209 shall I change this everywhere?

Reply:
@ShreyaPrasad1209 and me decided to use परियोजना earlier. so I used this in other modules too, shall I change this everywhere or only for mailers.

if the change is required I will create new pr as change will push different files for other modules too..

Reply:
@pavanjoshi914 let it be परियोजना for now, because we would have to change it in other files as well.